### ISSUE TITLE
Remove requirement to make vulnerabilities public (INC2)

### ISSUE THREAD

**Issue by EvansJonathan** [CVEProject/cveproject.github.io#11]
GOAL: Better define the correct use of CVE.
CHANGE: Should the requirement that CVE assignments must be public or planned to be made public be removed?
  - Should there be a new record type: "Not Public". This would accommodate the use case of a CVE ID being assigned without there being an intention for it to ever be made public (outside of the CNA's internal operation).

OUTCOME: CVE will be used as widely as necessary for as many use cases as practical.

**Comment by kurtseifried** [comment]
I agree and disagree with this. I would suggest more precise language like "CVE is for PUBLIC vulns right now" is moved to something like "CVE is for PUBLIC vulns and vulns shared between multiple organizations" perhaps.. obviously this needs a ton of word smithing. TBH I would personally prefer CVE be the bar code of vulns, public, private, one offs, whatever, but I do worry about the slippery slope of vuln info not being public/known.....

**Comment by EvansJonathan** [comment]
To me, this is a binary decision because we won't know how the private CVE IDs are being used or assigned, so we will not be able to enforce any restrictions on them.  In fact, none of the counting rules would be enforceable against the private assignments.

**Comment by ghost** [comment]
So some comments:

1) the private CVEs would still have to be reported as RESERVED
2) perhaps we have some requirement for a new state like "PRIVATE" if we adopt this, perhaps with some minimal amount of info like affected product/version
3) We already have a lot of vuln info not being known publicly, some CNAs are only doing CVEs for critical issues, or publicly found issues, so this problem already exists.
